src/server/tests.rs

```diff
@@ -3135,3 +3135,25 @@ async fn compute_diagnostics_multi_file() {
 
     assert_eq!(0, diagnostics_again.len());
 }
+
+#[test]
+async fn compute_diagnostics_multi_file_error_in_another_file() {
+    let server = create_server();
+
+    let filename: String = "file:///path/to/script.flux".into();
+    let fluxscript = r#"from(bucket: "my-bucket")
+|> range(start: -100d)
+|> filter(fn: (r) => r.anTag == v.a)"#;
+    open_file(&server, fluxscript.into(), Some(&filename)).await;
+    open_file(
+        &server,
+        r#"v = a"#.to_string(),
+        Some("file:///path/to/an_vars.flux"),
+    )
+    .await;
+
+    let diagnostics_again = server
+        .compute_diagnostics(&lsp::Url::parse(&filename).unwrap());
+
+    assert_eq!(0, diagnostics_again.len());
```

Comment:
The name of the test hints that we should be expecting an error (but only for one of the two files), yet the assertions check that no diagnostics are given.
The test prior is likely verifying that the diagnostic is actually surfaced correctly for the problem script.

I wonder if these tests should be rephrased to tell the whole story together? Maybe not -- maybe just tweaking the test name is best: "diagnostics only emitted by problem file" or something.